#!/usr/bin/env python
PACKAGE = "voronoi_planner"

from dynamic_reconfigure.parameter_generator_catkin import ParameterGenerator, int_t, double_t, bool_t

gen = ParameterGenerator()

gen.add("weight_data", double_t, 0, "Weight of data when smoothing path", 0.5, 0.1, 1)
gen.add("weight_smooth", double_t, 0, "Weight of smoothing when smoothing path", 0.3, 0.1, 1)

gen.add("publish_voronoi_grid", bool_t, 0, "Publish Voronoi Grid", True)
gen.add("smooth_path", bool_t, 0, "Smooth path", True)


exit(gen.generate(PACKAGE, "voronoi_planner", "VoronoiPlanner"))
